.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
Yes I know, it's complicated to show you a live sample. But I ensure you this will not change on tests integrity at all.

What did you not understand on my concrete cases? Or do you have another interrogations?